.github/workflows/build-and-test.yml

```diff
@@ -69,5 +69,5 @@ jobs:
     uses: ./.github/workflows/build-and-test-template.yml
     with:
       working_directory: ${{ matrix.working_directory }}
-      dotnet_version: ${{ vars.DOTNET_VERSION }}
+      dotnet_version: ${{ vars.DOTNET_VERSION_V8 }}
```

Comment:
The intention behind `DOTNET_VERSION` was that we could change it to 8.0.x when we did the .NET 8 upgrade (etc for further upgrades in the future), rather than add a new var

Reply:
Resolved, will be change back once we sync all release branches to this release :)